src/gui/search/searchwidget.cpp

```diff
@@ -162,6 +163,14 @@ bool SearchWidget::eventFilter(QObject *object, QEvent *event)
             closeTab(tabIndex);
             return true;
         }
+        else if (mouseEvent->button() == Qt::RightButton)
+        {
+            QMenu *menu = new QMenu(this);
+            menu->addAction(tr("Close tab"),this, [this, tabIndex](){ closeTab(tabIndex); });
```

Comment:
add space: `tr("Close tab"), this,`
and on the next line too.

Reply:
Thank you! One question: uncrustify, should i use it with config from root dir or it is outdated? I see several lines in this file that this utility tries to modify.  I do:  uncrustify -c ../../../uncrustify.cfg -f searchwidget.cpp  -o searchwidget_ucf.cpp and use kdiff3 after that.

Reply:
It is outdated and couldn't follow our coding style.